docs/source/dataset_script.rst

```diff
@@ -0,0 +1,360 @@
+                   "answers": datasets.features.Sequence(
+                       {"text": datasets.Value("string"), "answer_start": datasets.Value("int32"),}
+                   ),
+               }
+           ),
+           # No default supervised_keys (as we have to pass both question
+           # and context as input).
+           supervised_keys=None,
+           homepage="https://rajpurkar.github.io/SQuAD-explorer/",
+           citation=_CITATION,
+       )
+
+Multiple configurations
+^^^^^^^^^^^^^^^^^^^^^^^
+
+In some cases, your dataset may have multiple configurations. For example, the `SuperGLUE <https://huggingface.co/datasets/super_glue>`_ dataset is a collection of 5 datasets designed to evaluate language understanding tasks. 🤗 Datasets provides :class:`datasets.BuilderConfig` which allows you to create different configurations for the user to select from.
+
+Let's study the `SuperGLUE loading script <https://github.com/huggingface/datasets/blob/master/datasets/super_glue/super_glue.py>`_ to see how you can define several configurations.
+
+1. Create a :class:`datasets.BuilderConfig` class with attributes about your dataset. These attributes can be the features of your dataset, label classes, and a URL to the data files.
+
+.. code-block::
+
+   class SuperGlueConfig(datasets.BuilderConfig):
+       """BuilderConfig for SuperGLUE."""
+
+   def __init__(self, features, data_url, citation, url, label_classes=("False", "True"), **kwargs):
+       """BuilderConfig for SuperGLUE.
+
+       Args:
+       features: `list[string]`, list of the features that will appear in the
+           feature dict. Should not include "label".
+       data_url: `string`, url to download the zip file from.
+       citation: `string`, citation for the data set.
+       url: `string`, url for information about the data set.
+       label_classes: `list[string]`, the list of classes for the label if the
+           label is present as a string. Non-string labels will be cast to either
+           'False' or 'True'.
+       **kwargs: keyword arguments forwarded to super.
+       """
+       # Version history:
+       # 1.0.2: Fixed non-nondeterminism in ReCoRD.
+       # 1.0.1: Change from the pre-release trial version of SuperGLUE (v1.9) to
+       #        the full release (v2.0).
+       # 1.0.0: S3 (new shuffling, sharding and slicing mechanism).
+       # 0.0.2: Initial version.
+       super(SuperGlueConfig, self).__init__(version=datasets.Version("1.0.2"), **kwargs)
+       self.features = features
+       self.label_classes = label_classes
+       self.data_url = data_url
+       self.citation = citation
+       self.url = url
+
+2. Sub-class the base :class:`datasets.BuilderConfig` to add additional attributes of a configuration. This gives you more flexibility to specify the name and description of each configuration. These sub-classes should be listed under :obj:`datasets.DatasetBuilder.BUILDER_CONFIGS`:
```

Comment:
This is redundant with previous item. There we already had to subclass `BuilderConfig` in order to add additional attributes, like `features`, `label_classes` and `citation`. Note that the only attributes of base `BuilderConfig` are: `name`, `version`, `data_dir`, `data_files` and `description`.

Reply:
```suggestion
2. Sub-class the base :class:`datasets.BuilderConfig` to add additional attributes of a configuration. This gives you more flexibility to specify all the name and description of each configuration. These sub-class instances should be listed under :obj:`datasets.DatasetBuilder.BUILDER_CONFIGS`:
```

Reply:
I think this point is about creating the config instances and list them in BUILDER_CONFIGS.

Let me try to rephrase it